code/modules/urist/modules/uristjobs.dm

```diff
@@ -35,7 +35,7 @@
 		H.equip_to_slot_or_del(new /obj/item/clothing/gloves/combat(H), slot_gloves)
 		H.equip_to_slot_or_del(new /obj/item/clothing/shoes/jackboots(H), slot_shoes)
 		H.equip_to_slot_or_del(new /obj/item/clothing/under/rank/centcom(H), slot_w_uniform)
-		H.equip_to_slot_or_del(new /obj/item/clothing/suit/armor/vest/fluff/deus_blueshield(H), slot_wear_suit)
+		H.equip_to_slot_or_del(new /obj/item/clothing/suit/storage/vest(H), slot_wear_suit) //regular-ass armor vest for now
```

Comment:
changed it to a generic for now, switch it to whatever you feel fits; I don't think we need a custom armor now that accessories are unlimited.